code/modules/clothing/spacesuits/rig/suits/security.dm

```diff
@@ -1,3 +1,47 @@
+//Ishimura Sec RIGs
+
+
+//PCSI RIG
+/obj/item/weapon/rig/security/pcsi
+	name = "PCSI rig"
+	desc = "A lightweight and flexible armoured rig suit used by CEC shipboard security during crackdowns and for use in hazardous environments."
+	icon_state = "ishi_security_rig"
```

Comment:
this sprite is made for an advanced, heavier security rig, the PCSI rig should use a more basic one, this has been hashed out a lot recently